Data/Cleaned/README.md

```diff
@@ -0,0 +1,14 @@
+# Cleaned README
+
+Note: all information for mappings from measure columns and files to measure names can be found in Data/Metadata.xlsx. See Data README.md for more information. See Documentation folder for information on each measure.
+
+Folders in "Cleaned" are as follows:
+- 2021_release: contains cleaned files used in 2021_release, if they have been updated. Not updated files remain in main folder.
```

Comment:
I think this line could be a little clearer:

- 2021_release: contain older versions of cleaned files used in prior 2021 MWI release. Files without data updates remain in main folder.